src/cliAction/buildDeploy/handler_deploy.go

```diff
@@ -65,3 +76,45 @@ func (h *Handler) Deploy(ctx context.Context, config RunConfig) error {
 
 	return nil
 }
+
+func getConfigContent(config RunConfig) (*zeropsApiProtocol.StringNull, error) {
+	workingDir, err := filepath.Abs(config.WorkingDir)
+	if err != nil {
+		return nil, err
+	}
+
+	if config.ZeropsYamlPath != nil {
+		workingDir = path.Join(workingDir, *config.ZeropsYamlPath)
+	}
+
+	zeropsYamlPath := path.Join(workingDir, zeropsYamlFileName)
+
+	zeropsYamlStat, err := os.Stat(zeropsYamlPath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			if config.ZeropsYamlPath != nil {
+				return nil, errors.New(i18n.BuildDeployZeropsYamlNotFound)
+			}
+		}
+		return nil, nil
```

Comment:
Why we don't care about the error here?

Reply:
Because it's optional for api, so it doesn't make sense to force user to include one. Only if he specifies direct path to it.

Reply:
Ah I see, okay then.